Makefile

```
@@ -44,3 +44,8 @@ endif
 .PHONY:lint
 lint:
 	@$(POETRY_BIN) run ruff check
+
+.PHONY:docs_serve
+docs_serve:
```

```suggestion
.PHONY:docs-serve
docs-serve:
```

Since we're already using dashes, might as well continue